CIP-1904/README.md

```diff
@@ -0,0 +1,250 @@
+This data may be shared between all supply chain stakeholders as well as delivered to end customers.
+A producer is responsible to collect the SCM data associated with its product/good and insert it into the supply chain ecosystem in a defined manner specific to the use-case.
+
+We strongly recommend that the off-chain SCM schema follows an industry standard relevant to the particular use-case.
+For example, the Georgian Wine use-case from the Cardano Foundation with the National Wine Agency of Georgia uses an extended version of the [OIV Standard](https://www.oiv.int/standards/international-standard-for-the-labelling-of-wines) from the [International Organisation of Vine and Wine](https://www.oiv.int).
+Using such standards brings credibility to the transactions and helps foster enterprise adoption.
+
+##### Off-chain format
+Each verifiable record comes from a specific producer by `producerID`.
+An SCM batch can contain records from multiple producers, hence the off-chain data is a JSON map of `producerID` to an array of JSON objects.
+The structure of the JSON objects within the array is dependant on the subtype `st`.
+
+The `producerID` is a unique identifier for a producer in the supply chain ecosystem of the given subtype.
+We recommend short IDs if possible to save on space and costs.
+
+The following example contains 3 verifiable records from producer `a2gh` and producer `p33l`, where `{...}` represents an arbitrary JSON object for a given verifiable record:
+```
+{
+  a2gh: [{...}, {...}, {...}],
+  p33l: [{...}, {...}]
+}
+```
+
+##### On-chain format
+The following field labels are used within the on-chain metadata:
+
+|**Field**|**Description**|**Type**|**Set Value**|
+|---|---|---|---|
+|`t` for type|The type defines the structure of the data.|string|scm|
+|`st` for subtype|This **optional** field can be used to specify/describe a use-case. Please see the Subtypes section for more information.|string||
+|`v` for version|The version of the specification.|string|1|
+|`cid`|The Content Identifier of the off-chain data as defined by [Multiformats](https://github.com/multiformats/cid).|string||
+|`d` for verification data|Map of `producerID`s to their respective verfication information.|map||
+|`pk` for public key|The public key belonging to `producerID`. The field accepts the direct byte stream of the public key (embedded), or alternatively accepts a URL to the public key. The public key URL when resolved must serve the raw bytes using a HTTP Content-Type of `application/octet-stream`. Public keys must be verified by some predefined process relevant to the subtype. In case a particular producer performs a key rotation within a single transaction, `pk` can accept an array of URLs or byte streams. In this scenario, `h` must also be converted into an array of headers, and `s` must be converted into an array of arrays, where each top-level array represents all signatures for a given public key.|byte-stream or array of byte-streams||
```

Comment:
This is no longer limited to `Ed25519`. If it needs to be limited, the subtype can limit it.

For now, it's only limited broadly to what JSON Web Signatures support.